### DISCUSSION TITLE
How to access the older version?

### DISCUSSION THREAD

**Discussion by pnhuy** [MIT-LCP/mimic-code#1272] [MIMIC-III] [answered]
Hi guys,

I granted the permission to access the MIMIC-III data. 
The latest version is v.1.4, however I want to access the older one (v1.3) to reproduce a research.
Could you please tell me how to get the older version?

Thanks,
Huy

**Comment by alistairewj** [comment, marked as answer]
We don't typically distribute older versions of MIMIC after they've been taken down. Any research done on v1.3 should replicate on v1.4. Exact comparisons may not be possible, but the only major data change was the addition of negative cultures, so I imagine you should see mostly comparable results with v1.4.
